packages/toolpad-app/src/runtime/ToolpadApp.tsx

```diff
@@ -34,6 +34,8 @@ import {
   JsExpressionAttrValue,
 } from '@mui/toolpad-core';
 import { createProvidedContext, useAssertedContext } from '@mui/toolpad-utils/react';
+import { filterKeys, mapProperties, mapValues } from '@mui/toolpad-utils/collections';
+import { set as setObjectPath } from 'lodash-es';
```

Comment:
If we plan to eventually completely get rid of `lodash` we could maybe create utility methods for these operations, which for now would still use `lodash` but then we would know where to replace it.
Just a possibility, maybe it helps to have the `lodash` imports less spread out.

Reply:
To be honest, I'm not sure we should even have utility functions for this.

Reply:
Got it, yeah for this specific type of operation maybe just normal Javascript would be possible and be more type-safe.